.github/workflows/image.yml

```diff
@@ -8,154 +8,77 @@ on:
-
-      - name: Extract metadata for image
-        id: meta
-        uses: docker/metadata-action@902fa8ec7d6ecbf8d84d538b9b233a880e428804 # v5
-        with:
-          images: |
-            ghcr.io/${{ github.repository }}
-            ${{ github.repository }}
-          tags: |
-            type=sha,prefix=,format=long,suffix=-${{ matrix.platform }}
-          labels: |
-            org.opencontainers.image.title=${{ github.event.repository.name }}
-            org.opencontainers.image.description=${{ github.event.repository.description }}
-            org.opencontainers.image.vendor="Sentry"
-            org.opencontainers.image.revision=${{ github.sha }}
-            org.opencontainers.image.version=${{ github.sha }}
-            org.opencontainers.image.source=${{ github.server_url }}/${{ github.repository }}
-            org.opencontainers.image.url=${{ github.server_url }}/${{ github.repository }}
-
-      - name: Build & Push
-        id: build_push
-        uses: docker/build-push-action@32945a339266b759abcbdc89316275140b0fc960 # v6.8.10
-        with:
-          context: .
-          tags: ${{ steps.meta.outputs.tags }}
-          labels: ${{ steps.meta.outputs.labels }}
+          image_name: 'uptime-checker'
           platforms: linux/${{ matrix.platform }}
-          cache-from: type=gha,scope=${{ matrix.platform }}
-          cache-to: type=gha,mode=max,scope=${{ matrix.platform }}
-          annotations: ${{ steps.meta.outputs.annotations }}
-          provenance: mode=max
-          sbom: true
-          push: true
-
-  create-manifest:
-    name: Create multi-platform manifest
-    needs: build-main-or-release
-    if: |
-      github.repository_owner == 'getsentry'
-      && github.event_name == 'push'
-      && (github.ref == format('refs/heads/{0}', github.event.repository.default_branch) || startsWith(github.ref, 'refs/heads/release/'))
-
-    permissions:
-      contents: read
-      packages: write
-
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744 # v3
-
-      - name: Login to GHCR
-        uses: docker/login-action@74a5d142397b4f367a81961eba4e8cd7edddf772 # v3
-        with:
-          registry: ghcr.io
-          username: ${{ github.actor }}
-          password: ${{ secrets.GITHUB_TOKEN }}
-
-      - name: Login to Docker Hub
-        uses: docker/login-action@74a5d142397b4f367a81961eba4e8cd7edddf772 # v3
-        with:
-          username: sentrybuilder
-          password: ${{ secrets.DOCKER_HUB_RW_TOKEN }}
-
-      - name: Create and push multi-platform manifest
-        run: |
-          ./scripts/create-manifest.sh \
-            "${{ github.repository }}" \
-            "${{ github.sha }}" \
-            "${{ github.ref == format('refs/heads/{0}', github.event.repository.default_branch) }}"
+          dockerfile_path: './Dockerfile'
+          build_args: UPTIME_CHECKER_GIT_REVISION=${{ github.event.pull_request.head.sha || github.sha }}
```

Comment:
I see this in the dockerfile but don't see it in the previous code, what is it used for?